Sources/MlemMiddleware/Content Models/Community/Community2Providing.swift

```diff
@@ -64,25 +68,28 @@ public extension Community2Providing {
         }
     }
     
-    func toggleFavorite() {
+    @discardableResult
+    func toggleFavorite() -> Task<StateUpdateResult, Never> {
         updateFavorite(!favorited)
     }
     
-    func updateFavorite(_ newValue: Bool) {
+    @discardableResult
+    func updateFavorite(_ newValue: Bool) -> Task<StateUpdateResult, Never> {
         guard let subscriptions = self.api.subscriptions else {
             print("Tried to toggle favorite, but no SubscriptionList found!")
```

Comment:
*nit*

This should maybe be an `assertionFailure`? Not sure when we'd hit this unless something has gone wrong

Reply:
Sure. It's technically possible to hit it, but very unlikely, and `assertionFailure` won't cause a crash in the release build anyways